### ISSUE TITLE
SignerWithAddress in a different path vs "@nomiclabs/hardhat-ethers"

### ISSUE THREAD

**Issue by Jack-ZooFiLabs** [wighawag/hardhat-deploy-ethers#17]
So I followed the issue https://github.com/wighawag/hardhat-deploy-ethers/issues/12 to get `hardhat-deploy-ether` to work by adding an alias to `@nomiclab/ethers`

However, looks like `SignerWithAddress` is not in the same path so alias doesn't work perfectly. Please fix it.

The diff I have to make:
```
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signers";
```

**Comment by jeffreyzzzs** [comment]
Encountered the same problem, is there a solution?

**Comment by wighawag** [comment]
@Jack-ZooFiLabs @jeffreyzzzs I published v0.3.0-beta.13 that now reuse the same compilation output settings as hardhat-ethers

Let me know if this does not work (And reopen the issue). Thanks for the reports!
